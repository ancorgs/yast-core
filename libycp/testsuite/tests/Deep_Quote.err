Parsed:
----------------------------------------------------------------------
``(Time)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
``((8 + 5))
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
``(`Time)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
``(``(Time))
----------------------------------------------------------------------
