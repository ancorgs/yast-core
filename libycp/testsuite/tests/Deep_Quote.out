(Time)
((8 + 5))
(`Time)
(``(Time))
